### PR TITLE
Shadow Finesse

### PR DESCRIPTION
**Pull request by pianoblook**
Am I doing this right? lol we'll find out

### CODE REVIEW

File: Reference.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ If you are a brand new player, do not bother reading this document. It is much b
 If you have already read the beginners guide, then please see the [learning path document](Learning_Path.md). This explains what to learn next and how many games you should be playing before reading ahead.
 
 Note that convention updates for the group are listed in the `#announcements` channel on the [Hanabi Discord server](https://discord.gg/FADvkJp). (You can also review the [commit history](https://github.com/Zamiell/hanabi-conventions/commits/master) of this repository.)
-
+asdfasdfasdfasf
```
Comment (Zamiell) [inline Reference.md:10]: nope

Comment (pianoblook) [reply Reference.md:10]: what did I break

Comment (Zamiell) [reply Reference.md:10]: look at the line i am commenting on

Comment (pianoblook) [reply Reference.md]: ok try now? I think I fixed it

**Comment by Zamiell** [comment]
blind-play needs to be hyphenated
https://travis-ci.com/github/Zamiell/hanabi-conventions/builds/180287940

**Pull request merged**